airqo-platform/AirQo-api
# error logging hotfixes to device registry

**_WHAT DOES THIS PR DO?_**
error logging hotfixes to device registry

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [AN-110]

**_HOW DO I TEST OUT THIS PR?_**
```
cd src/device-registry
npm install
npm run stage-mac
```

Afterwards, just test out any endpoint with an invalid tenant name ---then observe message in the corresponding Slack channel.





[AN-110]: https://airqoteam.atlassian.net/browse/AN-110?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
